sophia/src/term/iri.rs

```diff
@@ -160,7 +160,7 @@ where
         }
     }
 
-    /// The length of this IRI.
+    /// The length in this IRI.
```

Comment:
Conceptually, the IRI is a single string, and this is the len *of* that string...